src/state/farms/fetchFarms.ts

```diff
@@ -82,11 +80,13 @@ const fetchFarms = async (farmsToFetch: FarmConfig[]) => {
 
       return {
         ...farmConfig,
-        tokenAmount: tokenAmount.toJSON(),
-        quoteTokenAmount: quoteTokenAmount.toJSON(),
+        tokenAmountMc: tokenAmountMc.toJSON(),
+        quoteTokenAmountMc: quoteTokenAmountMc.toJSON(),
+        tokenAmountTotal: tokenAmountTotal.toJSON(),
+        quoteTokenAmountTotal: quoteTokenAmountTotal.toJSON(),
```

Comment:
After these changes, only `tokenAmountTotal` is actually used elsewhere in the codebase. 

These aren't, but I think it's conceivable that these values would be useful? Maybe I leave them in for now but rip them out after finishing the pricing refactor if they still aren't useful?

`tokenAmountMc` - amount of base token staked in MC
`quoteTokenAmountMc` - amount of quote token staked in MC 
`quoteTokenAmountTotal` - amount of quote token in total